.github/workflows/api-e2e.yml

```diff
@@ -27,47 +27,64 @@ jobs:
   run-e2e-tests:
     defaults:
       run:
-        working-directory: ./Application/EdFi.Ods.Admin.Api/
+        working-directory: ./Application/EdFi.Ods.AdminApi/
 
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+
       - name: Update certificates
         run: cp -r ../../eng/test-certs/ssl ./Docker
+
       - name: Run Admin API
         run: |
-          docker-compose -f './Compose/pgsql/compose-build-dev.yml' --env-file './E2E Tests/gh-action-setup/.automation.env' up -d
-      - name: Verify ODS Health
+          docker-compose \
+            -f './Compose/pgsql/compose-build-dev.yml' \
+            --env-file './E2E Tests/gh-action-setup/.automation.env' \
+            up -d
+
+      - name: Verify containers are running
```

Comment:
This is a little confusing looking. I manually wrapped the old line 39 so that it would be easier to read. And I changed the next step's name from "Verify ODS Health" (which was not an accurate description) to "Verify containers are running".